### ISSUE TITLE
run bootstrap script on heroku deploy

### ISSUE THREAD

**Issue by stevecrozz** [errbit/errbit#1129]
Use https://devcenter.heroku.com/articles/app-json-schema#scripts to specify that heroku should run the bootstrap script automatically. I'm thinking it would be a good idea to combine this with environment variables that define the admin user and password so users don't have to go hunting for them.

**Comment by lucianosousa** [comment]
good idea. I was struggling here to login in a heroku fresh installation.
also, the documentation should be improved in that part

**Comment by rud** [comment]
There's a chicken/egg problem with how to reveal the root login to the installing user, as we have no secure channel to them aside from console output. I think we need to document where Heroku output can be seen after bootstrapping is completed, no?

**Comment by stevecrozz** [comment]
That's probably the easiest solution. We could also attempt to email you based on some heroku environment variable that may exist? Or try to use the heroku API to discover your email? Or have some kind of interactive initial setup page you can access when no users exist...

All of that would require a lot more effort and not sure we have the manpower for it. Probably the right thing to do is document where to find the output.
